ios/MaasAccount/Sources/MaasAccount/Requirements/Components/ToggleView.swift

```diff
@@ -0,0 +1,86 @@
+import SwiftUI
+import Swappable
+
+public struct ToggleView<ToggledContent: View, UntoggledContent: View>: View  {
+
+    @Binding var isOn: Bool
+
+    private let toggledContent: ToggledContent
+    private let untoggledContent: UntoggledContent
+
+    public init(
+        isOn: Binding<Bool>,
+        @ViewBuilder toggledContent: () -> ToggledContent,
+        @ViewBuilder untoggledContent: () -> UntoggledContent
```

Comment:
why not a single function with a bool?
```suggestion
        @ViewBuilder content: (Bool) -> ToggledContent,
```

Reply:
Because we have `Binding<Bool>` and we have separated contents when we change value. If i'm missing something lets make a call and discus it. :)